libc/test.yaml

```diff
@@ -1,10 +1,35 @@
 test:
   type: unity
   tests:
-    - name: unit
-      execute: test-libc
-      #FIXME: symlink tests will fail on non-rootfs targets
-      #FIXME: unistd_uid doesn't work on zynq7000-qemu
+    - name: printf
+      execute: test-libc-printf
       targets:
         exclude: [armv7m4-stm32l4x6-nucleo]
         include: [host-generic-pc]
+
+    # Executing test-printf alone on 'armv7m4-stm32l4x6-nucleo' takes ~4 min, that's why we want to run it only during nightly campaigns 
```

Comment:
```suggestion
    # Executing test-printf alone on 'armv7m4-stm32l4x6-nucleo' takes ~4 min, because of custom pexpect send method used on this target and large amount of test cases. 
    # That's why we want to run it only during nightly campaigns 
```

Reply:
Ok, I see that, not so many commands are sent in this test, so it shouldn't be the main factor.

Reply:
Yeah, sth disturbing happens there, maybe the process of loading a binary using `restore` command in Jlink takes a long time. @mateusz-bloch please investigate it before a merge.

Reply:
- the CI overhead is worrying and should be investigating further
- still, 4mins for the test is also quite a long time, we should probably investigate it further (can be done at a later time)

Reply:
Yeah, it's not a problem with this PR. In general it seems that all tests are executed much longer on stm target (for example comparing to imxrt106x), the results of investigation will be presented on Confluence.